test/UMTS.hs

```diff
@@ -47,6 +47,7 @@ import qualified Music.Score.Internal.Util
 import qualified Music.Pitch.Literal
```

Comment:
As per my comment on #21 I wouldn't bother with this file. If it's causing compiler errors feel free to remove it.